test.disabled/ejabberd_tests/tests/component_SUITE.erl

```diff
@@ -468,8 +468,8 @@ connect_component(ComponentOpts, StartStep) ->
     case Res of
     {ok, Component, _} ->
         {component, ComponentName} = lists:keyfind(component, 1, ComponentOpts),
-        {host, ComponentHost} = lists:keyfind(host, 1, ComponentOpts),
-        ComponentAddr = <<ComponentName/binary, ".", ComponentHost/binary>>,
+        {server, ComponentServer} = lists:keyfind(server, 1, ComponentOpts),
+        ComponentAddr = <<ComponentName/binary, ".", ComponentServer/binary>>,
         {Component, ComponentAddr, ComponentName};
```

Comment:
map is preferable as a return value (maybe not in this pr).

Reply:
Can be refactored as a part of "move to helper" PR.